packages/react-router/index.ts

```diff
@@ -254,15 +256,18 @@ export function createMemoryRouter(
   routes: RouteObject[],
   opts?: {
     basename?: string;
-    future?: FutureConfig;
+    future?: Partial<Omit<FutureConfig, "v7_prependBasename">>;
```

Comment:
`createMemoryRouter` does not accept `v7_prependBasename` since it always wants to prepend